html/janus.js

```diff
@@ -3482,19 +3438,17 @@ function Janus(gatewayCallbacks) {
 
 	function isDataEnabled(media) {
 		Janus.debug("isDataEnabled:", media);
-		if(Janus.webRTCAdapter.browserDetails.browser == "edge") {
+		if(Janus.webRTCAdapter.browserDetails.browser === "edge") {
 			Janus.warn("Edge doesn't support data channels yet");
 			return false;
 		}
-		if(media === undefined || media === null)
+		if(!media)
 			return false;	// Default
-		return (media.data === true);
+		return media.data;
```

Comment:
Users may have provided garbage in the `data` property: this should only return `true` if `data` was really `true`.

Reply:
Agree, and the check is the same for all of the `isEnabled` methods.

Reply:
Actually, not for *all* of them, since `audio` and `video` can be objects (e.g., to specify the deviceId) or strings (e.g., "screen").